src/Main.java

```diff
@@ -1,5 +1,53 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello, World!");
+
+        System.out.println("Задача№1");
+
+        int[] expenses = {1500, 2300, 1200, 800, 3000};
+        int total = 0;
+        for (int expense : expenses) {
+            total += expense;
+        }
+        System.out.println("Сумма трат за месяц составила " + total + " рублей");
+
+        System.out.println("Задача№2");
+
+        int[] expenses1 = {1500, 2300, 1200, 800, 3000};
+        int minExpense = expenses[0];
+        int maxExpense = expenses[0];
+        for (int expense : expenses) {
+            if (expense < minExpense) {
+                minExpense = expense;
+            }
+            if (expense > maxExpense) {
+                maxExpense = expense;
+            }
+        }
+        System.out.println("Минимальная сумма трат за неделю составила " + minExpense + " рублей. " +
+                "Максимальная сумма трат за неделю составила " + maxExpense + " рублей.");
+
+        System.out.println("Задача№3");
+
+        int[] expenses2 = {1500, 2300, 1200, 800, 3000};
+        int total1 = 0;
+        for (int expense : expenses) {
+            total1 += expense;
+        }
+        double average = total1 / 5.0;
```

Comment:
Не стоит "хардкодить" число 5 - в массиве может быть и 4 значения (в конкретном задании), а в реальность и 100 и 140 и любые другие, размер которых ты можешь не знать. Нужно брать размер массива из его свойств
```suggestion
        double average = total1 / expenses.length;
```